### ISSUE TITLE
Scrapper no longer needed

### ISSUE THREAD

**Issue by devadathanmb** [AJAYK-01/KTU-Notifier#22]
Hey there. KTU recently completely shifted their website to React and started making use of APIs for almost everything in their page. So this project might require a rewrite at some parts. Good thing is that we don't need to go through the pain of scraping anymore.

I have hacked together some code and tried to make a replica of this in typescript [(see here)](https://github.com/devadathanmb/ktu-bot) although I still haven't tested it out completely yet.

**Comment by AJAYK-01** [comment]
Interesting to see that KTU finally updated their website.

I am no longer a student of KTU, hence not actively updating this project. However any contributions to this project are welcome!

**Comment by devadathanmb** [comment]
They have indeed tried to make this website (the current one in react) live several times before this I guess but after a while they rolled back to the old version. Now I guess it's going to stay.

This project was basically my motivation for writing my own version for the updated website. I guess only the scrapper part needs to be rewritten and rest everything could stay.

**Comment by AJAYK-01** [comment]
> They have indeed tried to make this website (the current one in react) live several times before this I guess but after a while they rolled back to the old version. Now I guess it's going to stay.
> 
> This project was basically my motivation for writing my own version for the updated website. I guess only the scrapper part needs to be rewritten and rest everything could stay.

Yep, replacing scrapper with an api integration would make this work as expected. 

Also I checked out the bot you created, and I saw the ui and buttons for viewing notifications, which is possible due to the new api, so I think the api could further create more optimisations and better UX for this bot as well.
